### PR TITLE
docs: explain compression choices

### PR DESCRIPTION
**Pull request by blacha**

### CODE REVIEW

File: docs/compression.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Imagery Compression
+
+
+## Background
+
+Toitū Te Whenua (LINZ) holds multiple imagery types data for [most of New Zealand](https://www.linz.govt.nz/products-services/data/types-linz-data/aerial-imagery).
```
Comment (l0b0) [inline docs/compression.md:6]: > Page cannot be found (Error 404)

Comment (blacha) [reply docs/compression.md:6]: are you looking at this inside of LINZ? as the LINZ website doesn't seem to work well inside of LINZ?

File: docs/compression.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+Other types
+- 16bit 4band+ RGB+ satellite imagery - Not currently published
+
+
+### Goals
+
+As most of the imagery we are working on right now is RGB, the following document is mostly related to 8bit RGB imagery.
+
+LINZ uses GDAL for processing [COGs](https://gdal.org/drivers/raster/cog.html) which limits the compression types to LZW, DEFLATE, LERC, ZSTD and JpegXL
```
Comment (l0b0) [inline docs/compression.md:28]: "JpegXL" should be "JPEG XL"

File: docs/compression.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+```
+
+To ensure the imagery is lossless the imagery was converted back to a raw uncompressed imagery and then hashed
+
+```bash
+docker run \
+    -v tiffs-path:/input \
+    ghcr.io/osgeo/gdal:ubuntu-full-3.7.0 \
+    gdal_translate \
+        -of COG \
```
Comment (l0b0) [inline docs/compression.md:62]: Doesn't this contradict "uncompressed" in the paragraph above?

File: docs/compression.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        -co sparse_ok=true \
+    /output/${compression}_predictor-${precdictor}_level-${level}_error-${error}.tiff
+    /output/${compression}_predictor-${precdictor}_level-${level}_error-${error}-raw.tiff
+
+sha256sum /output/${compression}_predictor-${precdictor}_level-${level}_error-${error}-raw.tiff
+```
+
+## Results
+
+to start all tiffs were compressed as LZW to create a baseline
```
Comment (l0b0) [inline docs/compression.md:73]: It looks like this is missing some parts of the sentence.

File: docs/compression.md (hunk 1 above)
Comment (l0b0) [reply docs/compression.md:6]: Agh, this again!

Comment (blacha) [reply docs/compression.md:6]: Ill assume that means yes you are inside linz, as these links work fine for me.

File: docs/compression.md (hunk 2 above)
Comment (l0b0) [reply docs/compression.md]: There's still a couple of these.

**Review by dwsilk — APPROVED** [review]

**Review by amfage — APPROVED** [review]

**Pull request merged**